packages/app/src/app/store/modules/editor/actions.js

```diff
@@ -170,6 +170,23 @@ export function restartSandbox() {
   dispatch({ type: 'socket:message', channel: 'sandbox:restart' });
 }
 
+export function setupExecutor({ state, executor }) {
+  const sandbox = state.get('editor.currentSandbox');
+
+  return executor.initializeExecutor(sandbox).then(() => {
+    executor.listen('connect', 'server.onSSEMessage');
```

Comment:
@CompuIves All these `listen` call can be simplified, since the  second argument is always `server.onSSEMessage`.

```js
const events = ['connect', 'disconnect', 'sandbox:status', 'sandbox:start', 'sandbox:stop', 'sandbox:log', 'sandbox:hibernate', 'sandbox:update'];

events.forEach(event => {
  executor.listen(event, 'server.onSSEMessage');
});
```

Reply:
I like that! Will change it later